app/client/components/pages/singleBulkDisbursementPage.jsx

```diff
@@ -126,7 +141,20 @@ class SingleBulkDisbursementPage extends React.Component {
             </p>
             <p>
               {" "}
-              <b>Created by:</b> {createdBy || " "}
+              <b>Created by:</b>
+              <a
+                style={{ cursor: "pointer" }}
+                onClick={() =>
+                  this.navigateToUser(creator_user && creator_user.id)
+                }
+              >
+                {creator_user && " " + creator_user.email}
+              </a>
+            </p>
+            <p>
+              {" "}
+              <b>Approved by:</b>
```

Comment:
Currently shows `Approved By` even if the transfer was rejected. The easiest is to change to `Reviewed By`, or add a ternary operator to check if approved or rejected.

<img width="729" alt="Screen Shot 2021-04-27 at 10 20 34 am" src="https://user-images.githubusercontent.com/27039316/116175132-53c48400-a742-11eb-959c-4c59a41ca460.png">